origin-discovery/src/apollo/db.js

```diff
@@ -161,6 +162,10 @@ async function getListing (listingId, blockInfo = null) {
 function _makeOffer (row) {
   return {
     id: row.id,
+    blockInfo: {
+      blockNumber: row.data.events[0].blockNumber,
```

Comment:
This should aways work, just to improve error readability in case it faults I would add: 
```
if (row.data.events.length === 0 || row.data.events[0].event !== 'OfferCreated')
  throw new Error('Can not find OfferCreated event')

return {
    id: row.id,
    blockInfo: {
    ...
```

Reply:
Defensive coding is always a good idea. Thanks for the suggestion. Added.